### PR TITLE
Fix older blogs rendering

### PR DESCRIPTION
**Pull request by prabhanshuguptagit**
A bunch of old blogs have formatting that's broken/doesn't work well with the new blog's CSS.

- Tweaks layout for old blogs that weren't looking correct. None of the changes are content related.
- Better descriptions for some blogs
- Some spacing changes in CSS. Also fixes h2/h3/h4 rendering
- Adds missing images to the repo

### CODE REVIEW

File: source/_data/post_metadata.yml [hunk 1]
```diff
@@ -4,21 +4,21 @@
 
 - title: "Training, the Simple way"
   image: https://cdn-images-1.medium.com/max/1024/0*glZZfA4Axh6f9MeJ
-  description: "We experimented with several training methods: a short video combined with in-person support works best for us. One year ago (in October 2018), we piloted the Simple app in five public health facilities in Punjab, India. The original process of teaching healthcare workers to use Simple in pilot hospitals involved in-person trainings from members of our team. We flew from Bangalore to Bathinda, spent a week visiting one facility each day, and taught healthcare workers how to use the app. We’ve come a long way since then."
+  description: "One year ago (in October 2018), we piloted the Simple app in five public health facilities in Punjab, India. The original process of teaching healthcare workers to use Simple in pilot hospitals involved in-person trainings over a week from members of our team. We’ve come a long way since then."
```
Comment (prabhanshuguptagit) [inline source/_data/post_metadata.yml:7]: Descriptions updated.

File: source/_posts/2008-02-03-testing-strange-web-applications-using-selenium-take-note.markdown [hunk 2]
```diff
@@ -5,5 +5,5 @@ created_at: 2008-02-03 11:22:00 UTC
...
...ng the <span style="font-family: courier new;">onload </span>event, you'll need to reference <span style="font-family: courier new;">parent.selenium.browserbot</span> instead. As per the original solution, execute the following code anywhere (either statically in the page or in the <span style="font-family: courier new;">onload</span> event itself) before the JavaScript which pops up a dialog:<br /><br /><span style="font-family: courier new; font-weight: bold;">var browserbot = parent.selenium.browserbot;</span><br /><span style="font-family: courier new; font-weight: bold;"> if (browserbot) {</span><br /><span style="font-family: courier new; font-weight: bold;">    browserbot.modifyWindowToRecordPopUpDialogs(window, browserbot);</span><br /><span style="font-family: courier new; font-weight: bold;"> }</span><br /><br />Tada! You're home free and your QAs can happily automate testing through the UI once again.<br /><br />Also of note: If you're popping up alert and confirm dialogs in an <span style="font-family: courier new;">IFrame </span>(egads!), you'll need to reference <span style="font-family: courier new;">parent.parent.selenium.browserbot</span>.<br /><br />Happy testing!
```
Comment (prabhanshuguptagit) [inline source/_posts/2008-02-03-testing-strange-web-applications-using-selenium-take-note.markdown:8]: > font-size:180%
This is the only change here.

File: source/_posts/2018-07-11-fast-sudoku-solver-in-haskell-2-a-200x-faster-sol.markdown [hunk 3]
```diff
@@ -5,6 +5,7 @@ created_at: 2018-07-11 00:00:00 UTC
 author: Abhinav Sarkar
 post_url: "https://abhinavsarkar.net/posts/fast-sudoku-solver-in-haskell-2/"
 layout: post
+hidden: true
```
Comment (prabhanshuguptagit) [inline source/_posts/2018-07-11-fast-sudoku-solver-in-haskell-2-a-200x-faster-sol.markdown:8]: 3 haskell blogs were showing up on the grid. Hides all but one for variety. They'll still show up on the archive/elsewhere.

File: source/_posts/2008-02-03-how-to-be-succinct.markdown [hunk 4]
```diff
@@ -5,5 +5,12 @@ created_at: 2008-02-03 06:19:00 UTC
...
+Regarding Singletons, Steve Yegge writes:
+
+
+<blockquote><p> In my Data Structures course in college, when we got to AVL trees, my prof turned and wrote on the board, in huge, clear letters: </p>   <p><span style=";font-size:78%;color:red;"  ><pre> <span style="color: rgb(153, 0, 0);">AVL Trees are EVIL</span></pre></span> </p>   <p> ...and that's all we had to learn about them. He had us implement red/black trees and splay trees instead. To this day, I have no idea how threaded AVL trees work. But if that's OK with Dan Weld, it's OK with me. </p>  <p> But now I know how he felt.</p></blockquote>
+
+<span style="font-size:100%;">Steve, Steve. Follow Mr. Weld's example. Or you could just point people toward "Test-Driven Development By Example", since Mr. Beck has already covered this one pretty well:
+
+<blockquote>"How do you provide global variables in languages without global variables? Don't. Your programs will thank you for taking the time to think about design instead."</blockquote>
```
Comment (prabhanshuguptagit) [inline source/_posts/2008-02-03-how-to-be-succinct.markdown:15]: Example

Before
 

After

**Review by PriyangaPKini — APPROVED** [review]
LGTM

**Review by shhivam — APPROVED** [review]

**Pull request merged**